src/plugins/data/public/antlr/opensearch_ppl/ppl_documentation.ts

```diff
@@ -0,0 +1,1498 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// TODO: Add a script that scrapes the data to generate this file
```

Comment:
does this make sense to keep this in our repo? for me it's a little confusing that this would be housed in our repo because SQL and PPL functionality is derived from another repo. if someone updates the functionality in the SQL plugin will they be maintaining out repo documentation as well?